action/protocol/execution/evm/evm.go

```diff
@@ -371,8 +372,14 @@ func executeInEVM(evmParams *Params, stateDB *StateDBAdapter, g genesis.Blockcha
 	if evmErr != nil {
 		errCode = evmErrToErrStatusCode(evmErr, g, blockHeight)
 		if errCode == uint64(iotextypes.ReceiptStatus_ErrUnknown) {
+			var address string
+			if evmParams.contract != nil {
```

Comment:
when contract address is empty (""), will the contract field be nil?

Reply:
yes, empty "" --> evmParams.contract == nil